### PR TITLE
[Alert activations] Fix paginated_list_alert_activations and add test

### PR DESCRIPTION
**Pull request by rokatyy**
JIra - https://iguazio.atlassian.net/browse/ML-8483

Bug fix + new integration tests for project SDK (both regular one and paginated)
TODOs are there until https://iguazio.atlassian.net/browse/ML-8505 is done

### CODE REVIEW

File: tests/integration/sdk_api/alerts/test_alerts.py [hunk 1]
```diff
@@ -228,6 +228,92 @@ def test_alert_operations(self):
...
+        activations = project.list_alert_activations()
+        assert len(activations) == 2 * iterations
+
+        activations, token = project.paginated_list_alert_activations(
+            page_size=iterations
+        )
+        assert len(activations) == iterations
+
+        activations, token = project.paginated_list_alert_activations(page_token=token)
+        assert len(activations) == iterations
```
Comment (moranbental) [inline tests/integration/sdk_api/alerts/test_alerts.py:284]: This is the last page, so the token should be None here, right?

File: tests/integration/sdk_api/alerts/test_alerts.py [hunk 2]
```diff
@@ -228,6 +228,92 @@ def test_alert_operations(self):
...
+                self._post_event(project_name, event_name, alert_entity)
+
+        # check regular SDK without filters
+        activations = project.list_alert_activations()
+        assert len(activations) == 2 * iterations
+
+        activations, token = project.paginated_list_alert_activations(
+            page_size=iterations
+        )
+        assert len(activations) == iterations
```
Comment (moranbental) [inline tests/integration/sdk_api/alerts/test_alerts.py:280]: Verify here that the token is not None

File: tests/integration/sdk_api/alerts/test_alerts.py [hunk 3]
```diff
@@ -228,6 +228,92 @@ def test_alert_operations(self):
...
+
+        activations, token = project.paginated_list_alert_activations(page_token=token)
+        assert len(activations) == iterations
+
+        activations, token = project.paginated_list_alert_activations(page_token=token)
```
Comment (moranbental) [inline tests/integration/sdk_api/alerts/test_alerts.py]: I am not sure why you are sending this again, as the token is None here

File: tests/integration/sdk_api/alerts/test_alerts.py [hunk 4]
```diff
@@ -228,6 +228,92 @@ def test_alert_operations(self):
...
+                    page_token=token
+                )
+                assert len(activations) == 1
+                assert token is not None
+
+            activations, token = project.paginated_list_alert_activations(
+                page_token=token
+            )
+            assert len(activations) == 1
+            # TODO: uncomment when pagination is fixed
```
Comment (moranbental) [inline tests/integration/sdk_api/alerts/test_alerts.py:312]: Add the ticket number to the comment.

**Review by moranbental — CHANGES_REQUESTED** [review]
Looks good! just minor comments

File: tests/integration/sdk_api/alerts/test_alerts.py (hunk 1 above)
Comment (rokatyy) [reply tests/integration/sdk_api/alerts/test_alerts.py:284]: @moranbental right, sorry, that's leftovers from the tests I showed you recently, will remove this check

**Review by moranbental — APPROVED** [review]

**Review by liranbg — APPROVED** [review]

**Pull request merged**